homeassistant/components/smlight/sensor.py

```diff
@@ -30,11 +30,44 @@
 class SmSensorEntityDescription(SensorEntityDescription):
     """Class describing SMLIGHT sensor entities."""
 
-    entity_category = EntityCategory.DIAGNOSTIC
     value_fn: Callable[[Sensors], float | None]
 
 
-SENSORS = [
+@dataclass(frozen=True, kw_only=True)
+class SmInfoEntityDescription(SensorEntityDescription):
+    """Class describing SMLIGHT information entities."""
+
+    value_fn: Callable[[Info], int | str | None]
+
+
+INFO: list[SmInfoEntityDescription] = [
+    SmInfoEntityDescription(
+        key="device_ip",
+        translation_key="device_ip",
+        icon="mdi:ip-network-outline",
+        value_fn=lambda x: x.device_ip,
+    ),
+    SmInfoEntityDescription(
+        key="device_mode",
+        translation_key="device_mode",
+        icon="mdi:connection",
+        value_fn=lambda x: x.coord_mode,
+    ),
+    SmInfoEntityDescription(
+        key="firmware_channel",
+        translation_key="firmware_channel",
+        value_fn=lambda x: x.fw_channel,
+    ),
+    SmInfoEntityDescription(
+        key="zigbee_type",
+        translation_key="zigbee_type",
+        icon="mdi:zigbee",
+        value_fn=lambda x: x.zb_type,
+    ),
```

Comment:
From the description it sounds like all these entities have predefined values, please supply them in snake case in `options` and add translations for them in `strings.json`

Reply:
ok